common/src/main/java/com/microsoft/identity/common/internal/broker/ipc/IIpcStrategy.java

```diff
@@ -0,0 +1,21 @@
+package com.microsoft.identity.common.internal.broker.ipc;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.microsoft.identity.common.exception.BaseException;
+
+/**
+ * an interface for inter-process communication strategies.
+ */
+public interface IIpcStrategy {
```

Comment:
Can we give them a getName()?

Reply:
I'll make this as part of the ests telemetry work.